plugins/TimingTriggerCandidateMaker.cpp

```diff
@@ -40,8 +40,8 @@ TimingTriggerCandidateMaker::TimeStampedDataToTriggerCandidate(const triggeralgs
   triggeralgs::TriggerCandidate candidate;
   auto now = std::chrono::steady_clock::now();
 
-  candidate.time_start = data.time_stamp - m_map[data.signal_type].first,  // time_start
-  candidate.time_end = data.time_stamp + m_map[data.signal_type].second, // time_end,
+  candidate.time_start = data.time_stamp - m_detid_offsets_map[data.signal_type].first,  // time_start
```

Comment:
This will go wrong if `data.signal_type >= m_detid_offsets_map.size()`, so need to check for that case

Reply:
ok